app/models/user_password.rb

```diff
@@ -32,6 +27,6 @@ class UserPassword < ActiveRecord::Base
 # Indexes
 #
 #  idx_user_passwords_on_user_id_and_expired_at_and_hash  (user_id,password_expired_at,password_hash)
-#  index_user_passwords_on_user_id                        (user_id) UNIQUE WHERE (password_expired_at IS NULL)
+#  index_user_passwords_on_user_id                        (user_id) UNIQUE
 #  index_user_passwords_on_user_id_and_password_hash      (user_id,password_hash) UNIQUE
```

Comment:
I think we can drop this index now? Seems unnecessary since we have the unique index on user_id

(obviously not a big deal at this stage. But once we start migrating data into this table, fewer indexes might help with performance)